public/create-password/index.html

```diff
@@ -0,0 +1,228 @@
+                ></path>
+              </svg>
+              <!--<![endif]-->
+              <!--[if IE 8]>
+                <img
+                  src="/assets/images/govuk-logotype-crown.png"
+                  class="govuk-header__logotype-crown-fallback-image"
+                  width="36"
+                  height="32"
+                />
+              <![endif]-->
+              <span class="govuk-header__logotype-text"> GOV.UK </span>
+            </span>
+          </a>
+        </div>
+        <div class="govuk-header__content">
+          <a href="#" class="govuk-header__link govuk-header__service-name">
+            Birmingham City Council
+          </a>
+        </div>
+      </div>
+    </header>
+
+    <div class="govuk-width-container">
+      <main class="govuk-main-wrapper" id="main-content" role="main">
+        <a href="/email-address" class="govuk-back-link" data-cy="back-link"
+          >Back</a
+        >
+        <span id="error-summary"></span>
+
+        <form
+          action="/create-password"
+          method="POST"
+          autocomplete="off"
+          novalidate="true"
+          spellcheck="false"
+          id="account-password"
+        >
+          <div id="password-fields">
+            <div class="govuk-form-group">
+              <h1 class="govuk-label-wrapper" data-cy="section-heading-h1">
+                <label class="govuk-label govuk-label--l" for="password">
+                  Create a password for your account
+                </label>
+              </h1>
+              <div id="password-name-hint" class="govuk-hint">
+                Password must contain 8 digits, at least one number and one
+                special character
+              </div>
+              <span id="error"></span>
+              <input
+                class="govuk-input govuk-input--width-20 password"
+                id="password"
+                name="password"
+                type="password"
+                data-cy="password-input"
+              />
+              <br />
```

Comment:
Was already on the template? If not, it might be better to adjust the spacing of elements with CSS.